src/org/labkey/test/LabKeySiteWrapper.java

```diff
@@ -986,9 +986,13 @@ public void goToProjectSettings(String project)
         goToProjectSettings();
     }
 
+    /**
+     * @deprecated Use {@link org.labkey.test.LabKeySiteWrapper#goToAdminConsole}
+     */
+    @Deprecated
```

Comment:
Set `forRemoval` to encourage prompt removal.
```suggestion
    @Deprecated (forRemoval = true)
```